tests/unit/slider/methods.js

```diff
@@ -109,8 +109,54 @@ test( "value", function() {
 	equal( element.slider( "value" ), 460, "value is restricted to maximum valid step" );
 });
 
-//test( "values", function() {
-//	ok(false, "missing test - untested code is broken code." );
-//});
+test( "values", function() {
+	expect( 12 );
+	
+	var element = $( "<div></div>" ).slider({
+		range: true,
+		min: 10,
+		max: 100,
+		step: 1
+	});
+
+	element.slider( "values", [ 20, 90 ] );
+	deepEqual( element.slider( "values" ), [ 20, 90 ], "Values (array) - get value for all handles" );
+
+	element.slider( "values", [ 0, 200 ] );
+	deepEqual( element.slider( "values" ), [ 10, 100 ], "Values (array) - restricted min and max" );
+
+	element.slider( "values", 0, 50 );
+	equal( element.slider( "values", 0 ), 50, "Values (index,value) - set first handle" );
+
+	element.slider( "values", 1, 75 );
+	equal( element.slider( "values", 1 ), 75, "Values (index,value) - set second handle" );
+
+	
+	element.slider( "values", 0, 5 );
+	equal( element.slider( "values", 0 ), 10, "Values (index,value) - 1st handle restricted properly, against min" );
+
+	element.slider( "values", 0, 90 );
+	equal( element.slider( "values", 0 ), element.slider( "values", 1 ), "Values (index,value) - 1st handle restricted properly, against 2nd handle" );
```

Comment:
I understand that you're trying to show that the handles are matching, but I'd prefer an explicit value for the expected value.